tests/checker_test.go

```diff
@@ -0,0 +1,135 @@
+package tests
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/GoogleCloudPlatform/functions-framework-go/funcframework"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	handler "github.com/tablelandnetwork/basin-storage"
+)
+
+func buildCheckerRequest(t *testing.T) *http.Request {
+	urlStr := fmt.Sprintf("http://localhost:%s", functionsPort)
+	data := url.Values{}
+	data.Set("simulated", "true")
+	req, err := http.NewRequest("POST", urlStr, strings.NewReader(data.Encode()))
+	require.NoError(t, err)
+
+	req.Header.Add("Content-Type", "application/x-www-form-urlencoded")
+	return req
+}
+
+func TestChecker(t *testing.T) {
```

Comment:
integration test for the check that uses an old job that has deals already created for it